UnityProject/Assets/Scripts/Systems/Antagonists/Objectives/EscapeObjectives/Escape.cs

```diff
@@ -51,7 +51,7 @@ protected override bool CheckCompletion()
 
 			return !Owner.Body.playerHealth.IsDead &&
 				ValidShuttles.Any( shuttle => shuttle.MatrixInfo != null
-					&& (CheckOnShip(Owner.Body.RegisterPlayer, shuttle.MatrixInfo.Matrix)) && shuttle.HasWorkingThrusters);
```

Comment:
Might be worth checking if   the player is in x range of central command?, If this is just a quick fix then this is fine